catt/cli.py

```diff
@@ -103,28 +105,98 @@ def write_config(settings):
         raise CattCliError("No device specified.")
 
 
+def hunt_subtitle(video):
+    dot_pos = video.rfind(".")
```

Comment:
It would be nice to use Pathlib here, and also to write a more robust algorithm. My subtitle names aren't always `videofile.srt`, sometimes they are `videofile.en.srt` and `videofile.el.srt`, so it would be good to do some globbing and fetch the first file alphabetically, if nothing is selected.